### PR TITLE
GAWB-1019: added last modified column to workspace list

### PR DESCRIPTION
**Pull request by skwalker**
- [x] **Submitter**: Include the JIRA issue number in the PR description
- [x] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [x] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * Add notes on what you've tested
- [x] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [x] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [x] **Submitter**: If you're adding new libraries, sign us up to security updates for them
- [x] Tell the tech lead (TL) that the PR exists if they wants to look at it
- [x] Anoint a lead reviewer (LR). **Assign PR to LR**
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
- [x] **TL** sign off
- [x] **LR** sign off
- [x] **Product Owner** sign off
- [x] **Assign to submitter** to finalize
- [x] **Submitter**: Verify all tests go green, including CI tests
- [x] **Submitter**: Squash commits and merge to develop
- [x] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

**Comment by MattPutnam** [comment]
👍 LGTM.  Do we want to put this info on the summary page too?

**Comment by skwalker** [comment]
Yeah, I was wondering that too.. It's not part of the ticket but let me see what Moran says

**Comment by davidangb** [comment]
works for me. When you talk to Moran, also ask if we should use moment.js to display human-friendly dates like "1 day ago" instead of the full datetime - relegate the datetime to a tooltip.

**Comment by skwalker** [comment]
Talked to Moran, leaving as is (with date instead of "1 day ago", and only in workspace list, not in summary). Last commit fixes the tooltip.

File: src/cljs/org/broadinstitute/firecloud_ui/page/workspaces_list.cljs [hunk 1]
```diff
@@ -162,6 +162,10 @@
                                  (if description (-> description split-lines first)
                                                  [:span {:style {:fontStyle "italic"}}
                                                   "No description provided"])])}
+           {:as-text (fn [date] (common/format-date date))
```
Comment (MattPutnam) [inline src/cljs/org/broadinstitute/firecloud_ui/page/workspaces_list.cljs]: `(fn [date] (common/format-date date))` -> `common/format-date`

File: src/cljs/org/broadinstitute/firecloud_ui/page/workspaces_list.cljs [hunk 2]
```diff
@@ -162,6 +162,10 @@
...
+            :header "Last Modified Date" :starting-width 300 :marginLeft 14
```
Comment (MattPutnam) [inline src/cljs/org/broadinstitute/firecloud_ui/page/workspaces_list.cljs]: `:marginLeft` doesn't do anything here

**Pull request merged**